website/content/docs/other-specifications/acl-policy.mdx

```diff
@@ -110,6 +110,7 @@ Each namespace rule can include a coarse-grained `policy` field, a fine-grained
 `capabilities` field, a `variables` block, or all three.
 
 The `policy` field for namespace rules can have one of the following values:
+- `deny` - prevent any capabilities
```

Comment:
Oops, isn't this as dupe of the `deny` below?

Reply:
What I read from the code is that `deny` can be used as both a fine-grained capability or a top level coarse-grained capability. I'm not sure if I am miss-understanding how these work together.

```hcl
namespace "default" {
  policy       = "deny"
  capabilities = ["deny"]
}
```

The "important" bit of the [code I was reading](https://github.com/hashicorp/nomad/blob/a9d61ea3fd9ecb51d64534b30fe5938024336787/acl/policy.go#L262):
```
case PolicyDeny:
  return []string{NamespaceCapabilityDeny}
```